TTYC.Application/Products/AddProduct/AddProductHandler.cs

```diff
@@ -18,7 +19,15 @@ public AddProductHandler(ApplicationDbContext dbContext, IMapper mapper)
 
         public async Task<Guid> Handle(AddProductCommand command, CancellationToken cancellationToken)
         {
+            var stores = new List<Store>();
+            foreach (var id in command.StoreIds)
```

Comment:
You can use something like var stores = await dbContext.Stores.Where(x => command.StoreIds.Containes(s => x.Id)).ToList();